openapi/goto.meeting/Dependencies.toml

```diff
@@ -0,0 +1,311 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Shall we add the normal Dependencies.toml instead of this AUTO-GENERATED FILE?

Reply:
Resolved